docusaurus/sidebars-ios.json

```diff
@@ -1,49 +1,436 @@
 {
-    "docs": [
-      "home",
-      "introduction",
-      {
-        "Guides": [
-          "guides/integration",
-          "guides/getting-started",
-          "guides/working-with-channel-list",
-          "guides/working-with-a-channel",
-          "guides/working-with-messages",
-          "guides/working-with-user",
-          "guides/pinned-messages",
-          "guides/working-with-attachments",
-          "guides/working-with-reactions",
-          "guides/working-with-members",
-          "guides/working-with-watchers",
-          "guides/events",
-          "guides/push-notifications",
-          "guides/connection-status",
-          "guides/ui-customization",
-          "guides/moderation",
-          "guides/filter-query-guide",
-          "guides/multi-tenancy",
-          "guides/debugging"
-        ]
-      },
-      { "StreamChat Controllers":
-        [
-          {
-           "type": "autogenerated",
-           "dirName": "controllers"
-          }
-         ] 
-       },
-      
-      { "UI Components": 
-        [
-         {
-          "type": "autogenerated",
-          "dirName": "ui-components"
-         }
-        ] 
-      },
-      "migrating-from-1.x-and-2.x",
-      "faq",
-      "glossary"
-    ]
-  }
+  "docs": [
+    "home",
+    "introduction",
```

Comment:
Let's remove this and name the group below "Introduction" instead

Reply:
Done.